config/environments/production.rb

```diff
@@ -60,6 +60,7 @@
 
   # Use a different cache store in production.
   # config.cache_store = :mem_cache_store
+  config.cache_store = :redis_cache_store, { url: ENV.fetch("REDIS_URL") }
 
```

Comment:
This was what was missing last time! I couldn’t figure out where to get the key from, great finding!!!